addon-test-support/create.js

```diff
@@ -56,6 +57,30 @@ function buildChainObject(node, blueprintKey, blueprint, defaultBuilder) {
   return buildObject(node, blueprintKey, blueprint, defaultBuilder);
 }
 
+// collections are doubly complicated. First, they contain page object-like 
+// properties (ie properties created by the `create` function) that aren't truly
+// user created page objects. They also rely on Ceibo's `setup` function to delay 
+// the creation of the `collection` object to tree building time (rather than 
+// when defined), instead returning a Ceibo descriptor during defintion time.
+// They unavoidably rely on closure scoping to capture the descriptor which 
+// is mutated with a `value` property. Simply storing a converted definition
+// and always reinvoking the `collection` function before `create` is called 
+// avoids unintended shared state 
+function recreateCollections(definition){
```

Comment:
Thanks for confirming!

Well, my original intention to keep it un-merged was not to pollute v1.15(which is already quite full) with non-critical, but potentially harmful refactorings Unfortunatelly cutting a stable release of v1.15 hanged longer than I expected.

I think since there are no any user reports about any issues related to the current collection `setup` we can leave it un-merged some more time...

Meanwhile I think you can cherry-pick it to your branch to avoid the issue you have faced.